internal/enums/pres_action_type.go

```diff
@@ -2,10 +2,10 @@ package enums
 
 /*
 ENUM(
-Unspecified
-CreateAIP
-CreateAndReviewAIP
-MovePackage
+unspecified
+create aip
+create and review aip
+move package
```

Comment:
Just noticed, these were using hyphens: https://github.com/artefactual-sdps/enduro/pull/975/files#diff-5d31faeec1ed87c9e54c8a9f758ddbd6640bd2e1a116020c6466dfcdeb25ef28L264

Could that be an issue?

Reply:
It's not an issue for the Enduro or the Dashboard. The strings "create-aip", "create-and-review-aip", and "move-package" don't occur anywhere in this branch anymore.  I've tested the Dashboard and the "Create AIP" and "Move Package" titles shows up correctly on the package show page.  As far as I know the "Create and Review AIP" workflow is non-functional in Enduro.

![image](https://github.com/artefactual-sdps/enduro/assets/626135/3f4d258c-2ad6-4571-add0-403275046656)